portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -30,6 +30,7 @@
 import com.google.appengine.api.datastore.PreparedQuery;
```

Comment:
Please give PRs such as this one more descriptive names.

Reply:
Noted